bin_nisse/parse_qc_for_cdm.py

```diff
@@ -214,7 +214,7 @@ def calculate_het_qcs(het_call_vcf) -> Tuple[str, Dict[str, Any]]:
         "non_calls": non_calls,
         "nbr_het_calls": nbr_het_calls,
         "het_calls": calls,
-        "het_calls_fraction": nbr_het_calls / nbr_calls,
+        "pct_het_calls": round((nbr_het_calls / nbr_calls) * 100,1),
```

Comment:
I would prefer not rounding here and doing that in CDM instead, so we have the full number stored

Reply:
```suggestion
        "pct_het_calls": (nbr_het_calls / nbr_calls) * 100,
```